docs/config.rst

```diff
@@ -1,11 +1,623 @@
+
+See `cumulusci.yml reference`_ for a complete list of values that can be used in each section.
+
+
+
+Task Configurations
+-------------------
+
+Task configurations are defined under the ``tasks`` section of your project's ``cumulusci.yml`` file. Each task configuration defines a task which can be run using the ``cci task run`` command, or included in a flow step. With a few simple changes to this section, you can configure build automation functionality without ever touching Python code.
+
+
+Overriding a Task Option
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you continually specify the same value for an option while running a task, you can configure CumulusCI to use that particular value when running a task.
+
+    Example: You want to enforce a 90% code coverage requirement for Apex code in your project. The `run_tests <TODO>`_ task, which executes all Apex Tests in a target org, can enforce code coverage at a given percentage by passing the `required_org_code_coverage_percent` option.
+
+.. code-block:: yaml
+
+    run_tests:
+        options:
+            required_org_code_coverage_percent: 90
+
+When the code is placed under the ``tasks`` section of the project config file, CumulusCI overrides the default option with a value of ``90``. The overridden option will be used whenever this automated task is called, unless it is further overridden for a particular flow step.
+
+Verify the change by looking for a default option value when running ``cci task info <name>``:
+
+.. code-block:: yaml
+
+    $ cci task info run_tests
+    run_tests
+
+    Description: Runs all apex tests
+
+    Class: cumulusci.tasks.apex.testrunner.RunApexTests
+
+    Command Syntax
+
+        $ cci task run run_tests
+
+    Options
+        .
+        .
+        .
+      -o required_org_code_coverage_percent PERCENTAGE
+        Optional
+        Require at least X percent code coverage across the org following the test run.
+        Default: 90
+
+
+Add a Custom Task
+^^^^^^^^^^^^^^^^^
+
+To define a new task for your project, simply add the name of the new task under the ``tasks`` section of your ``cumulusci.yml`` file.
+
+    Example: In reference to the ``run_tests`` task above, create a custom task named ``deploy_reports`` to create a detailed report about what happens when ``run_tests`` is deployed.
+
+First, look up the Python class associated with the standard task ``deploy``. 
+
+[SCREENSHOT? CODE BLOCK?]
+
+We can see above that the ``deploy`` task has a ``class_path`` value of ``cumulusci.tasks.salesforce.Deploy``. Let's assume that reports are a part of your package's unpackaged metadata inside the ``unpackaged/config/reports`` folder.
+
+Add the following under the ``tasks`` section of the project config.
+    
+    .. note:: Be sure to include the value we retrieved for ``class_path``!
```

Comment:
One issue here is that ``note`` puts the text in a box, ``attention`` does not.

Reply:
@justinvholt - When previewing the doc in github you're correct: this admonition isn't recognized, so it doesn't render properly. But the actual end-user version of the documentation (which is rendered through Sphinx using a custom theme) does recognize. Here is a[ list ](https://pradyunsg.me/furo/reference/admonitions/#supported-types) of the supported admonitions that we can use with the theme.

Because of this extra "rendering" step, we typically include a link to the [rendered docs](Rendered Docs: https://cumulusci.readthedocs.io/en/feature-doc-reorg__config/config.html) at the top of each pull request so that we can view what the final version of the docs will look like.